ts/src/htx.ts

```diff
@@ -6874,19 +6874,19 @@ export default class htx extends Exchange {
             const status = this.safeString (response, 'status');
             if (status === 'error') {
                 const code = this.safeString2 (response, 'err-code', 'err_code');
-                const feedback = this.id + ' ' + body;
-                this.throwBroadlyMatchedException (this.exceptions['broad'], body, feedback);
-                this.throwExactlyMatchedException (this.exceptions['exact'], code, feedback);
                 const message = this.safeString2 (response, 'err-msg', 'err_msg');
-                this.throwExactlyMatchedException (this.exceptions['exact'], message, feedback);
-                throw new ExchangeError (feedback);
+                const errorMessage = code + ': ' + message;
+                this.throwBroadlyMatchedException (this.exceptions['broad'], body, errorMessage);
+                this.throwExactlyMatchedException (this.exceptions['exact'], code, errorMessage);
+                this.throwExactlyMatchedException (this.exceptions['exact'], message, errorMessage);
+                throw new ExchangeError (errorMessage);
             }
         }
         if ('code' in response) {
             // {code: '1003', message: 'invalid signature'}
-            const feedback = this.id + ' ' + body;
             const code = this.safeString (response, 'code');
-            this.throwExactlyMatchedException (this.exceptions['exact'], code, feedback);
+            const message = this.safeString (response, 'message');
```

Comment:
Could you please share the doc where the error has `message` attribute? Didn't find this

Reply:
I do not know if it's somewhere in the documentation but I trusted the guy who left the comment 2 lines above:
`// {code: '1003', message: 'invalid signature'}`